src/elements/logo/logo.visual.spec.ts

```diff
@@ -22,7 +22,9 @@ describe(`sbb-logo`, () => {
     describeEach(cases, ({ negative, protectiveRoom }) => {
       beforeEach(async function () {
         root = await visualRegressionFixture(
-          html` <sbb-logo ?negative=${negative} protective-room=${protectiveRoom}></sbb-logo> `,
+          html` <div style="max-width: 300px">
```

Comment:
could you also check my PR, I moved the maxWidth into the wrapper (I remember other examples where it could be used). and another question, the viewport could now be set to zero as zero is 320px and max-width is 300px?

Reply:
Nice, I'll wait for when you merge your changes

Reply:
It's merged, you can go ahead ;)

Reply:
Done